sycl/test-e2e/InvokeSimd/Spec/ImplicitSubgroup/uniform_retval.cpp

```diff
@@ -1,6 +1,3 @@
-// TODO: enable when Jira ticket resolved
-// XFAIL: gpu
```

Comment:
I'm afraid it will enable Win as well, and there was no update there. Need to wait for the CI to finish before merging.

Reply:
We don't run these on Windows, see [here](https://github.com/intel/llvm/blob/sycl/sycl/test-e2e/InvokeSimd/lit.local.cfg)